rmt-helm/README.md

```diff
@@ -34,17 +34,18 @@ paths handling, as NGINX is configured to do that.
 ## Custom mandatory values
 
 Certain values of the chart do not have any defaults:
-- SCC mirroring credentials (refer to [more information](https://documentation.suse.com/sles/15-SP4/html/SLES-all/cha-rmt-mirroring.html#sec-rmt-mirroring-credentials) for more information)
+- SCC mirroring credentials (refer to [more information](https://documentation.suse.com/sles/html/SLES-all/cha-rmt-mirroring.html#sec-rmt-mirroring-credentials) for more information)
 - list of products to mirror
 - list of products not to mirror
 - DNS name used to reach the RMT server
 - configured [storage](https://kubernetes.io/docs/concepts/storage/)
 
 Before deploying the chart, you must fill a custom values file.
 
-The following example enables ingress with TLS.
-The create-certs.sh can be used to create self-signed certificates and
-add them to Kubernetes as a usable TLS secret.
+The following example enables ingress with TLS. The create-certs.sh script
+(available in the [helm-charts GitHub repository](https://github.com/SUSE/helm-charts/tree/main/rmt-helm)) can be used
```

Comment:
same like above. the create-certs.sh script comes with the chart (however you obtained it) so this isn't necessary.